### PR TITLE
autopep8: add page

### PR DESCRIPTION
**Pull request by Waples**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/autopep8.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# autopep8
+
+> Automatically formats Python code to confrom to the PEP 8 style guide.
+> More information: <https://github.com/hhatto/autopep8>.
+
+- Print out formatted file with set maximium allowed line length:
+
+`autopep8 {{path/to/file}} --max-line-length {{len}}`
+
+- Print the diff for the fixed source:
```
Comment (bl-ue) [inline pages/common/autopep8.md:10]:
```suggestion
- Format a file, displaying a diff of the changes:
```

File: pages/common/autopep8.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# autopep8
+
+> Automatically formats Python code to confrom to the PEP 8 style guide.
+> More information: <https://github.com/hhatto/autopep8>.
+
+- Print out formatted file with set maximium allowed line length:
```
Comment (bl-ue) [inline pages/common/autopep8.md:6]:
```suggestion
- Format a file to stdout, with a custom maximium line length:
```

File: pages/common/autopep8.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`autopep8 {{path/to/file}} --max-line-length {{len}}`
+
+- Print the diff for the fixed source:
+
+`autopep8 --diff {{path/to/file}}`
+
+- Format the file in-pace:
```
Comment (bl-ue) [inline pages/common/autopep8.md:14]:
```suggestion
- Format a file in-pace:
```

File: pages/common/autopep8.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`autopep8 --in-place {{path/to/file}}`
+
+- Format the files in-pace recursivly in dir:
```
Comment (bl-ue) [inline pages/common/autopep8.md:18]:
```suggestion
- Recursively format all files in a directory in-place:
```

File: pages/common/autopep8.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# autopep8
+
+> Automatically formats Python code to confrom to the PEP 8 style guide.
```
Comment (bl-ue) [inline pages/common/autopep8.md:3]:
```suggestion
> Format Python code according to the PEP 8 style guide.
```

File: pages/common/autopep8.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/hhatto/autopep8>.
+
+- Print out formatted file with set maximium allowed line length:
+
+`autopep8 {{path/to/file}} --max-line-length {{len}}`
```
Comment (bl-ue) [inline pages/common/autopep8.md:8]:
```suggestion
`autopep8 {{path/to/file.py}} --max-line-length {{len}}`
```

File: pages/common/autopep8.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print the diff for the fixed source:
+
+`autopep8 --diff {{path/to/file}}`
+
+- Format the file in-pace:
+
+`autopep8 --in-place {{path/to/file}}`
```
Comment (bl-ue) [inline pages/common/autopep8.md:16]:
```suggestion
`autopep8 --in-place {{path/to/file.py}}`
```

File: pages/common/autopep8.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Format the files in-pace recursivly in dir:
+
+`autopep8 --in-place --recursive {{path/to/dir}}`
```
Comment (bl-ue) [inline pages/common/autopep8.md:20]:
```suggestion
`autopep8 --in-place --recursive {{path/to/directory}}`
```

File: pages.nl/common/autopep8.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# autopep8
+
+> Formatteert automatisch Python-code conform de PEP 8-stijlgids.
+> Meer informatie: <https://github.com/hhatto/autopep8>.
+
+- Print het geformatteerde bestand af met een ingestelde maximale toegestane regellengte:
+
+`autopep8 {{pad/naar/bestand}} --max-line-length {{len}}`
```
Comment (bl-ue) [inline pages.nl/common/autopep8.md:8]:
```suggestion
`autopep8 {{pad/naar/bestand.py}} --max-line-length {{len}}`
```

File: pages.nl/common/autopep8.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print de diff voor de het geformatteerde bestand:
+
+`autopep8 --diff {{pad/naar/bestand}}`
```
Comment (bl-ue) [inline pages.nl/common/autopep8.md:12]:
```suggestion
`autopep8 --diff {{pad/naar/bestand.py}}`
```

File: pages.nl/common/autopep8.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+- Formatteer het bestand en sla dit op:
+
+`autopep8 --in-place {{pad/naar/bestand}}`
```
Comment (bl-ue) [inline pages.nl/common/autopep8.md:16]:
```suggestion
`autopep8 --in-place {{pad/naar/bestand.py}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Welcome back @Waples!  I left some comments below.  Please also update the Dutch translation when you've applied the suggestions to the English one.

**Comment by Waples** [comment]
@bl-ue changes made.

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
@Waples thank you.  [Here](https://github.com/tldr-pages/tldr/pull/5645#discussion_r604255392) however, I updated the page description, and in d3970373d3ee077ed840236a3f44d75d2f2b9a53 I don't see an update to the nl one—can you update that real quick as well?

**Comment by Waples** [comment]
@bl-ue change made

File: pages.nl/common/autopep8.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
+# autopep8
+
+> Formatteer Python-code conform de PEP 8-stijlgids.
+> Meer informatie: <https://github.com/hhatto/autopep8>.
+
+- Formateer een bestand naar stdout, met een ingestelde maximale toegestane regellengte:
+
+`autopep8 {{pad/naar/bestand.py}} --max-line-length {{len}}`
```
Comment (bl-ue) [inline pages.nl/common/autopep8.md:8]:
```suggestion
`autopep8 {{pad/naar/bestand.py}} --max-line-length {{lengte}}`
```

**Comment by Waples** [comment]
nittpicker :p @bl-ue changes made

**Review by bl-ue — APPROVED** [review]
Thank you @Waples!

**Pull request merged**